data/processing/input_latency.html

```diff
@@ -51,10 +51,11 @@
 
   function inputLatencyMapper(result, model) {
     const results = [];
-
+    results.push(model.canonicalUrl)
```

Comment:
The output looks like this currently:
{
  "failures": [],
  "pairs": {
    "Latencies": [ "01cd6194e72e118c56234261c579c25a", 
                            { "Latency Duration": 100,
                              "Queue Events Bottom-Up": { "event" : duration .....}, 
                               .......
                            } ]
                  }
}

Each result object has a list of failures and an object of result 'pairs'. Latencies is a list of different latency events and the data associated with them as well as the URL. I can certainly separate the URL key out into a separate metadata object like you suggested, not sure if that really simplifies the data processing script but it will probably seem cleaner.